graylog2-web-interface/src/stores/tools/ToolsStore.js

```diff
@@ -0,0 +1,166 @@
+// @flow strict
+import Reflux from 'reflux';
+
+import fetch from 'logic/rest/FetchProvider';
+import ApiRoutes from 'routing/ApiRoutes';
+import URLUtils from 'util/URLUtils';
+import UserNotification from 'util/UserNotification';
+
+const ToolsStore = Reflux.createStore({
```

Comment:
Adding this store may be problematic, since 3.1. still had the old Typescript `ToolStore.ts` (see https://github.com/Graylog2/graylog2-server/pull/6766 for the PR doing that change).

I think it should be possible to keep this new JS version of the store and remove the old Typescript store, but we should ensure that only one of them is there, otherwise I'm not sure which one the import is going to get.